### PR TITLE
Save formatted phone numbers

### PR DESCRIPTION
**Pull request by estellecomment**
Issue #1770

### CODE REVIEW

**Comment by mandric** [comment]
@estellecomment fyi the original issue here needs the tagging and assignment for code review.  Think of the PR as a child of the issue.  See https://github.com/medic/medic-docs/blob/master/md/dev/workflow.md#with-pull-request

**Comment by estellecomment** [comment]
Thanks for the pointer @mandric! Should be all good now (apart from branch having the wrong name, sorry)

File: tests/karma/unit/enketo/phone-widget.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
+describe('phone-widget', function() {
```
Comment (mandric) [inline tests/karma/unit/enketo/phone-widget.js]: 4 or 2 space indentation would be nice in this module.

Comment (alxndrsn) [inline tests/karma/unit/enketo/phone-widget.js]: I would strongly recommend following the enketo code formatting in enketo widgets - most of the code tends to be boilerplate, and comparing the boilerplate with upstream won't be easy if the formatting's different.

**Comment by mandric** [comment]
@estellecomment did you push your changes to this branch?  Was looking for jquery.clone changes but don't see them.

**Comment by estellecomment** [comment]
Not yet. I added the catch() and that ballooned into all sorts of testing issues, so need to fix my tests first. I'll let you know.

**Comment by mandric** [comment]
@estellecomment  okay you can skip the catch if it's a pain, I looked at some other uses of SettingsP and in most of them we don't catch exceptions... so maybe just open an issue to try to fix all instances of that.  Also in this case the negative impact is minor, in acceptance testing we can check.  For example if SettingsP service fails we get no form as opposed to bad data?  I didn't want to block merging this since it's been sitting around for a while.  It's fine to open more issues if needed.

File: tests/karma/unit/enketo/phone-widget.js (hunk 1 above)
Comment (mandric) [inline tests/karma/unit/enketo/phone-widget.js]: So @alxndrsn is the enketo formatting standard to use 4 spaces?  I'll add a comment to the module.

Comment (alxndrsn) [inline tests/karma/unit/enketo/phone-widget.js]: It's encoded at https://github.com/enketo/enketo-core/blob/master/.jsbeautifyrc; basically 4 space indent, and a single space both sides of parentheses, unless they're empty in which cases there is no space inside the parens.  E.g.
```js
define( function( require, exports, module ) {
    'use strict';
    var FormModel = require( 'enketo-core/src/js/Form-model' );
    var x = function () {};
```

Comment (alxndrsn) [inline tests/karma/unit/enketo/phone-widget.js]: PR at https://github.com/medic/medic-webapp/pull/1791 to revert to previous style

**Pull request merged**